src/services/user.js

```diff
@@ -226,6 +226,7 @@ const createUser = async (req, res, next) =>
             languagesToTeach: languagesToTeach,
             languagesToLearn: languagesToLearn,
             userIsActivie: userIsActivie,
+            isActivated: false,
```

Comment:
This is still shown as a change :)

Reply:
for local, it was true to run the program which I firstly commit by mistake. Now I have correct it with another commit that's why it is showing as a change.

Reply:
But the point is, if it shows then this is different than your target branch. You can check this file on your local by checking out to devel branch or using the github file explorer ui.

Reply:
https://github.com/Tandem-App-Tuni/Kivako-app-backend/blob/devel/src/services/user.js#L228

The key `isActivated` does not exist. You have added it. Why?